ngx-tools/src/is-array-of-type/is-array-of-type.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * Helper function to determine if all the members of an array are of a given type
+ *
+ * @param x - array to test for type
+ * @param guard - test for specific type
+ * @return true or throw error if not correct type
+ */
+export function isArrayOfType<T>(x: any[], guard: (y: any) => y is T): x is T[] {
+  for (const value of x) {
+    if (!guard(value)) {
+      throw Error(`Items is not correct type: ${value}`);
```

Comment:
I think this is how core was using it originally, but I'm thinking we should probably just return true/false rather than throw an error. That way it handles values the same way our other coercion functions do.

So really, just `return guard(value)` (and update jsdoc)